imagersite/imagerprofile/templates/photo.html

```diff
@@ -0,0 +1,57 @@
+{% extends "base.html" %}
+
+{% block main %}
+
+  <script type="text/javascript" src="//code.jquery.com/jquery-2.1.4.min.js"></script>
+  <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/d3/3.5.6/d3.min.js"></script>
+  <script type="text/javascript" src="//algorithmia.com/v1/clients/js/algorithmia-0.2.0.js"></script>
+  <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/sweetalert/1.0.1/sweetalert-dev.js"></script>
+  <script src="/static/facetagger.js" type="text/javascript"></script>
+  <link rel='stylesheet' type='text/css' href="/static/facetagger.css"/>
```

Comment:
working on the assumption that your `main` block is inside the `<body>` of your html, this means that all these js tags and css links are as well.  That's not good.  You want to put js and css in the `<head>` of a document.  It will work here, but it's not good.